src/Containers/TopLandingSection/index.scss

```diff
@@ -0,0 +1,18 @@
+.top-landing-section{
+    &__container{
+        &--title{
+            font-size: 3vh;
+            margin-bottom: 8vh;
+        }
+        &--slogan{
+            font-size: 4vh;
+            margin-bottom: 20vh;
+            padding-right: 4vw;
+        }
+        &--description{
+            font-size: 3vh;
+            margin-bottom: 5vh;
+            padding-right: 4vw;
```

Comment:
I don't think it's a good idea to have font size depending on the size of a screen, and even less the height.

Reply:
@dusmel This is crucial for big screens or with projectors, the text will scale accordingly, making the text much more comfortable. I can use the viewport but it won't be as great as the one above

Reply:
@gracelungu imagine how big the font will be on those vertical screens. Having the viewport meta tag in the head will handle most of this case.

Reply:
@dusmel You can try it and you will see. and keep in mind to  use the right ratios